pom.xml

```diff
@@ -48,8 +48,8 @@
         <avro.version>1.9.2</avro.version>
         <required.maven.version>3.2</required.maven.version>
         <confluent.version>6.0.0-SNAPSHOT</confluent.version>
-        <kafka.version>6.0.0-ccs-SNAPSHOT</kafka.version>
-        <ce.kafka.version>6.0.0-ce-SNAPSHOT</ce.kafka.version>
+        <kafka.version>[6.0.0-0-ccs, 6.0.1-0-ccs)</kafka.version>
+        <ce.kafka.version>[6.0.0-0-ce, 6.0.1-0-ce)</ce.kafka.version>
```

Comment:
I'm going to be doing the branch cut on Monday, I have yet to see any enhancements to `release-tools` that will start replacing these values with ranges. Therefore I think I'm going to ask that you hold of shipping this PR until I've done that work on Monday, at which point, I think you'd want to apply this to a `6.0.x` branch vs master.

I'm mainly just trying to call out that theres needs to be a sync up here between @sdanduConf and myself with respect to timing, timelines, and LoE into `release-tools`.

Reply:
@andrewegel yes i will be rebasing and merging this to the 6.0.x branch. I have been coordinating the roll out of this with srini and will let you know when we have date set. As of now I just need to get the PR's reviewed before scheduling it.